tests/app/questionnaire/test_questionnaire_schema.py

```diff
@@ -5,6 +5,63 @@
 
 from app.questionnaire.questionnaire_schema import AnswerDependent, QuestionnaireSchema
 
+WHEN_RULES_SECTION_DEPENDENCIES_SCHEMA = {
+    "sections": [
+        {
+            "id": "skip-confirmation-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "skip-confirmation",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "skip-confirmation-answer",
+                                        "options": [
+                                            {"label": "No", "value": "No"},
+                                        ],
+                                    }
+                                ],
+                                "id": "skip-confirmation-question",
+                                "type": "General",
+                            },
+                        },
+                    ],
+                    "id": "skip-confirmation-group",
+                }
+            ],
+        },
+        {
+            "id": "confirmation-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "confirmation-question",
+                            "question": {},
+                        }
+                    ],
+                    "id": "confirmation-group",
+                    "skip_conditions": {
+                        "when": {
+                            "==": [
+                                {
+                                    "source": "answers",
+                                    "identifier": "skip-confirmation-answer",
+                                },
+                                "No",
+                            ]
+                        },
+                    },
+                },
+            ],
+        },
+    ]
+}
```

Comment:
Given we have gone down the road of already loading in existing schemas for the other dependency tests, we should do the same here and avoid putting schema in the code.

Reply:
I tried but conftest.py would exceed the 1200 lines limit so had to put it here...

Reply:
We should load in an existing schema not put json in that is basically invalid anyway. i.e you can't have exmpty questions etc. Have a look at the existing test for dependency which uses load schema method.

Reply:
Added reference to existing schema, thanks!